### PR TITLE
multiple pages: Add Korean Translation

### PR DESCRIPTION
**Pull request by dmakth**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ko/common/csvgrep.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# csvgrep
+
+> csvkit에 포함된 CSV행의 문자열 및 패턴 매칭 필터링. 
+> 더 많은 정보: <https://csvkit.readthedocs.io/en/latest/scripts/csvgrep.html>.
+
+- 1 열에 특정 문자열이 있는 행 찾기:
+
+`csvgrep -c {{1}} -m {{찾을 문자열}} {{데이터.csv}}`
```
Comment (Proscream) [inline pages.ko/common/csvgrep.md]: If you replace space writing with "_", LGTM

File: pages.ko/common/crystal.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`crystal build {{경로/파일.cr}}`
+
+- 언어 테스트를 위한 로컬 대화형 서버 시작:
+
+`crystal play`
+
+- Cystal 응용 프로그램을 위한 프로젝트 디렉토리 생성:
+
+`crystal init app {{어플리케이션 명}}`
```
Comment (Proscream) [inline pages.ko/common/crystal.md]: "어플리케이션명" 이 더 나을 듯!

Comment (dmakth) [reply pages.ko/common/crystal.md]: Thanks @Proscream

**Comment by sbrl** [comment]
Merging, since we have 2 approvals - and @Proscream's approval here ought to count.

**Pull request merged**